oak_tensorflow_service/build.rs

```diff
@@ -26,4 +32,27 @@ fn main() {
     // As long the generated client code is only used in tests it won't be
     // included in any binaries.
     oak_idl_gen_services::compile_services_clients(SCHEMA);
+
+    build_tflite();
+}
+
+/// Builds TensorFlow Lite static library and adds the corresponding build
+/// directory to the library search path.
+fn build_tflite() {
+    let status = Command::new("bazel")
+        .arg("build")
+        .arg(TFLITE_BUILD_TARGET)
+        .status()
+        .expect("Failed to run bazel build");
+    if !status.success() {
+        panic!("Failed to run bazel build: exit status is {}", status);
+    }
+
+    // Add TensorFlow Lite build directory to the library search path.
+    println!("cargo:rustc-link-search={}", TFLITE_LIBRARY_DIR);
+
+    // Rerun build.rs if TensorFlow Lite library has been changed.
+    let library_path =
+        PathBuf::from(TFLITE_LIBRARY_DIR).join(format!("lib{}.a", TFLITE_LIBRARY_NAME));
+    println!("cargo:rerun-if-changed={}", library_path.display());
```

Comment:
This seems strange. AFAICT it will rerun build.rs if the output of the bazel build changed. I assume this should be rerun if any of the TFLite source code changed.

Reply:
Done
Added `println!("cargo:rustc-link-search={}", TFLITE_LIBRARY_DIR.display());`